src/flamegraph/svg.rs

```diff
@@ -1,4 +1,5 @@
 use super::{Direction, Options};
+use enquote::enquote;
```

Comment:
It's a little tempting to just bring in [the function](https://docs.rs/enquote/1.0.3/src/enquote/lib.rs.html#19-35) directly rather than adding an entire external dependency for something this simple..